documentation/docs/installation.mdx

```diff
@@ -60,89 +71,16 @@ revealEnv.publicPath = `/reveal-worker/`;
 const viewer = new Cognite3DViewer(/*...*/)
 ```
 
-### Option 2: host Web Worker and WebAssembly files externally
-
-In case if there is no such `/public` or similar folder in your project, you're likely using a different service
-to host static assets. The domain of that service is probably whitelisted in Content Security Policy and can be used to host the necessary files.
-
-In that case, the only thing you can do is build the reveal-parser-worker from sources with correct `publicPath`.
-
-:::note
-
-**If you're developing a Cognite application,
-you likely can use our own whitelisted CDN where we already have
-[parser-worker](https://cdf-hub-bundles.cogniteapp.com/dependencies/@cognite/reveal-parser-worker/1.1.0/reveal.parser.worker.js)
-uploaded.**
-
-In that case, skip to the [last step](#lastCDNstep) for this option and use
-
-```js
-import { revealEnv } from '@cognite/reveal'
-revealEnv.publicPath =
-    `https://cdf-hub-bundles.cogniteapp.com/dependencies/@cognite/reveal-parser-worker/1.1.0/`
-```
-
-Parser-worker version can be different,
-see the latest versions on [npm page](https://www.npmjs.com/package/@cognite/reveal-parser-worker).
-:::
-
-Let's say your static server has that URL `https://static.server/`. And you can upload the files
-at some folder, so files should be available at `https://static.server/parser-worker/`
+  <details>
+    <summary>Other copying options</summary>
 
-So all you need to do is:
-
-1. Clone or download [@cognite/reveal](https://github.com/cognitedata/reveal).
-2. `cd ./reveal/parser-worker`
-3. Consult `parser-worker/README.md` and install the necessary build tools
-4. You will need to set `PUBLIC_PATH` env variable pointing to your static server,
-then run the build. You will see `publicPath` printed in console during the build
-
-<Tabs
-  groupId="operating-systems"
-  defaultValue="bash"
-  values={[
-    {label: 'Bash', value: 'bash'},
-    {label: 'Windows', value: 'win'},
-  ]
-}>
-<TabItem value="bash">
-
-```bash
-PUBLIC_PATH="https://static.server/parser-worker/" yarn build:prod
-```
-</TabItem>
-<TabItem value="win">
+Below are few more convenient options for copying if you don't like
```

Comment:
Don't like what?

Reply:
it has the code snippet right after the sentence, but I rephrased it a bit

![image](https://user-images.githubusercontent.com/4989157/106604952-a0902880-6560-11eb-9496-94e6fa2016fd.png)

looks ok?